src/simulation_loop.py

```diff
@@ -1,22 +1,24 @@
 import numpy as np
-import vision.QAK.reading_data as rd
+import vision.oak.reading_data as rd
 import map2d.translator as m2d
 
 if __name__ == "__main__":
-    base_path = "./data1500/void_1500-47/stairs0"
+
+    base_path = "../data1500/void_1500-47/stairs0"
     visualize = False
     S3D = rd.Simulation3D(base_path=base_path, visualize=visualize)
 
     res = 200
     map_2d = np.full((res, res), np.nan)
     value_range = (-1, 1)
     # visualization_type - 0 means using open3d, any other number means using regular matplotlib
```

Comment:
do tego warto zrobić klasę która przypisuje liczbom nazwy:
```python
class VizualizationType:
    OPEN3D = 0
    MATPLOTLIB = 1
```

Reply:
oooo nice  ale jako osobne issue bo to jest w kilki miejscach

Reply:
machnąłem: issue #37